kotlin-analysis-api/src/main/kotlin/com/google/devtools/ksp/impl/symbol/kotlin/KSPropertyDeclarationImpl.kt

```diff
@@ -239,3 +233,11 @@ internal fun KaPropertySymbol.toModifiers(): Set<Modifier> {
     }
     return result
 }
+
+internal fun KSAnnotation.isValidOnProperty(): Boolean =
+    annotationType.resolve().declaration.annotations.none { metaAnnotation ->
+        metaAnnotation.annotationType.resolve().declaration.qualifiedName?.asString() == "kotlin.annotation.Target" &&
+            (metaAnnotation.arguments.singleOrNull()?.value as? ArrayList<*>)?.none {
+            (it as? KSClassDeclaration)?.qualifiedName?.asString() == "kotlin.annotation.AnnotationTarget.PROPERTY"
+        } ?: false
```

Comment:
In the code review, it's hard to understand where this elvis operator belongs...

I wonder if the aligned indentation would help? IIUC, this is:
```kt
  (metaAnno...)?.none {
    (it as? ...)
  } ?: false
```
but it's currently:
```kt
  (metaAnno...)?.none {
  (it as? ...)
} ?: false
```
?

Reply:
I agree. This is a bug of ktlint.